DependencyInjection/PayumExtension.php

```diff
@@ -38,7 +39,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         // load services
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-        $loader->load('payum.xml');
+        $loader->load(Kernel::MAJOR_VERSION === 3 ? 'payum3x.xml' : 'payum.xml');
```

Comment:
It introduces a lot of duplications. 

It could be better to do something like this:

```php
<?php
$loader->load('payum.xml');
if (Kernel::MAJOR_VERSION === 3) {
  $loader->load('payum3x.xml');
}
```

`payum3x.xml` should contain only fixed services